modules/mobile/app/controllers/mobile/v0/immunizations_controller.rb

```diff
@@ -9,6 +9,17 @@ def index
         render json: Mobile::V0::ImmunizationSerializer.new(immunizations_adapter.parse(service.get_immunizations))
       end
 
+      def location
+        lh_location = service.get_location(params[:id])
+        id_string = lh_location[:identifier][0][:value]
+        id = id_string.slice(4, id_string.length)
```

Comment:
You could also do this without passing in the length `id_string[4..-1]` or `lh_location[:identifier].first[:value][4..-1]`